starknet_py/net/account/account.py

```diff
@@ -229,6 +228,16 @@ async def get_balance(
 
         return (high << 128) + low
 
+    async def sign_for_fee_estimate(
+        self, transaction: TypeAccountTransaction
+    ) -> TypeAccountTransaction:
+        version = self.supported_transaction_version + 2**128
```

Comment:
Should we add a constant?

Reply:
Also we should test it.

Reply:
And add the docstring

Reply:
Docstring is already there in the `BaseAccount`.

Reply:
I don't want to define a constant for this, this is Account implementation detail.